sw/cheri/sim_boot_stub/boot_sram.S

```diff
@@ -0,0 +1,16 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+	.section .text.start, "ax", @progbits
+.zero 0x80
+
+	.globl start
+	.p2align 2
+    .type start,@function
+start:
+  // Enable the ICache
+  csrsi            0x7c0, 1
+  auipcc           ct1, 0
```

Comment:
Perhaps a comment here to explain what it is doing ('jump to ...')

Reply:
Thanks, added.